### ISSUE TITLE
Pickup Optimizations

### ISSUE THREAD

**Issue by El1zabethS** [DoF-6413/chargedUp#71]
Create code so the end effector can intake cones and cubes on one button using the color sensor

**Comment by J-Palomino** [comment]
I think you are super close on this one, even if the arm can't safely hold cubes inside yet maybe there is a placeholder automation we can make and run on that button until the mechanical improvements have been made

**Comment by El1zabethS** [comment]
Yes! This one should actually be completed? It was working, but was taken out for AZ West because the color sensor would continuously come unplugged and the robot would default to cubes :/ When it functions and all mechanical components are in place, it should go through the correct processes for intaking a cube v cone
